05libraries/sec05Summary.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Summary
+---
+
+## Summary
+
+### No Magic Answer
+
+Managing libraries in C++ is more tricky than other languages.
```

Comment:
```suggestion
Managing libraries in C++ is trickier than other languages.
```
This feels like a discouraging statement, and is perhaps not justified in the broad context of programming (yes C++ libs take more work than python and some more recent languages, but others are also a pain.)